<launch>

	<node pkg="map_compression" type="compareDistribution" name="compareDistribution" output="screen">

	<param name="loadMapName1" value="/home/yh/mapModel/2019/01.08/learn/results/predict_b500_from_1000_test_compressed.ply"/>
	<param name="loadMapName2" value="/home/yh/mapModel/2019/01.08/learn/results/predict_b500_test_compressed.ply"/>
	<param name="saveDisName" value="/home/yh/mapModel/2019/01.08/learn/results/predict_b500_DIFF.txt"/>

  </node>
</launch>
